src/plugins/irc/irc-config.c

```diff
@@ -3227,6 +3246,30 @@ irc_config_init ()
         NULL, NULL, NULL,
         &irc_config_change_color_item_nick_modes, NULL, NULL,
         NULL, NULL, NULL);
+    irc_config_color_item_tls_version_max = weechat_config_new_option (
```

Comment:
I would rename this variable `irc_config_color_item_tls_version_ok` as it's for a TLS version that is **not** deprecated.